src/utils/IVeriCredential.ts

```diff
@@ -143,6 +143,7 @@ export interface Address {
 }
 
 export interface DrivingPermit {
+	documentType?: string;
```

Comment:
will it be a valid object if `documentType` is undefined?

Reply:
I don't know actually, that's not something I tested I don't think. Passport and DrivingPermit were like this already, which I replicated for ResidencePermit and IdentityCard (possibly wrongly thinking this was already addressed)